horace_core/sqw/coord_transform/@ortho_proj/ortho_proj.m

```diff
@@ -11,13 +11,21 @@
     %   >> proj = ortho_proj(proj_struct)
     %             where proj_struct is the
     %             structure, containing any fields, with names, equal any
-    %             public fields of the ortho_proj class
+    %             public fields of the ortho_proj class.
+    %
+    % As a standard serializable class accepts full set of positional and
```

Comment:
```suggestion
    % As a standard serializable class, class ortho_proj accepts full set of positional and
```
I'm adding text to make this more meaningful but I might have guessed the meaning wrong